src/lib/y2users/config.rb

```diff
@@ -67,6 +73,13 @@ def groups
       groups_collection.dup.freeze
     end
 
+    # Whether the login is configured
```

Comment:
I think this is root of my confusion. I think something like `Whether the login configuration is defined` make it clear that it is not about configure login, but if that configuration is defined or unknown.